cluster/certificates.go

```diff
@@ -269,7 +269,10 @@ func (c *Cluster) ExtractBackupCertificateBundle(ctx context.Context) error {
 	for _, host := range backupHosts {
 		runHost := host
 		errgrp.Go(func() error {
-			return pki.ExtractBackupBundleOnHost(ctx, runHost, c.SystemImages.Alpine, services.EtcdSnapshotPath, c.PrivateRegistriesMap)
+			if err := pki.ExtractBackupBundleOnHost(ctx, runHost, c.SystemImages.Alpine, services.EtcdSnapshotPath, c.PrivateRegistriesMap); err != nil {
```

Comment:
@galal-hussein could you explain the nature of the fix? We don't care if the extract fails on all the backup hosts, and just log it in warn?

Reply:
yes, there are some user cases where the ssl are already deployed on the machines somehow, for example:
- 1 etcd
- 1 control 
- 1 worker 

and you replace the etcd server, then in this case the certificates are already there on the control plane and in the next run it will be extracted normally

Reply:
@galal-hussein `in the next run it will be extracted normally` what if the next run doesn't happen? Lets say rancher calls create/update on a cluster, and etcd backup fails by some other reason but `ssl are already deployed on the machines somehow`? The create/update call will succeed, and the cluster will remain missing etcd backup till the next update - which might not happen in days